UnitTests/CardPaymentsTests/CardClient_Tests.swift

```diff
@@ -5,6 +5,8 @@ import AuthenticationServices
 @testable import CardPayments
 @testable import TestShared
 
+// swiftlint:disable type_body_length
+// swiftlint:disable file_length
```

Comment:
I think we can do this: 
```suggestion
// swiftlint:disable type_body_length, file_length
```

Reply:
It is strange, I still got error body length when I applied your suggestion.

Reply:
I don't think you need the comma, should work with `// swiftlint:disable type_body_length file_length`

Reply:
Thanks Jax!